fluentd-forward/README.md

```diff
@@ -0,0 +1,350 @@
+# Teleport-fluentd-forward
+
+This plugin is used to export Audit Log events to Fluentd service.
+
+## Setup
+
+### Prerequisites
+
+This guide assumes that you have:
+* Teleport 6.2 or newer
+* Admin privileges to access tctl
+* Docker to build plugin from source and run fluentd example instance
+
+The required Fluentd version for production setup is v1.12.4 or newer. Lower versions do not support TLS.
+
+## Installing the plugin
+
+We recommend installing the Teleport Plugins alongside the Teleport Proxy. This is an ideal location as plugins have a low memory footprint, and will require both public internet access and Teleport Auth access. We currently only provide linux-amd64 binaries, you can also compile these plugins from source.
+
+### Install the plugin from source
+
+Please ensure that Docker is running!
+
+```sh
+git clone https://github.com/gravitational/teleport-plugins.git --depth 1
+cd teleport-plugins/fluentd-forward/build.assets
+make install
+```
+
+This command will build `build/teleport-fluentd-forward` executable and place it to `/usr/local/bin` folder. The following error means that you do not have write permissions on target folder:
+
+```sh
+cp: /usr/local/bin/teleport-fluentd-forward: Operation not permitted
+```
+
+To fix this, you can either set target folder to something listed in your `$PATH`:
+
+```sh
+make install BINDIR=/tmp/test-fluentd-setup
+```
+
+or copy binary file manually with `sudo`:
+
+```sh
+sudo cp build/teleport-fluentd-forward /usr/local/bin
+```
+
+In sections below we assume that you have `teleport-fluentd-forward` executable available in your `$PATH` or cwd.
+
+## Generate example configuration
+
+Run:
+
+```sh
+teleport-fluentd-forward configure .
```

Comment:
Done. Mock arg for now. It might require different cli args for different targets in future, so, possibly it would be better to rename `configure` to `configure-fluentd`?